website/content/api-docs/secret/openldap.mdx

```diff
@@ -216,36 +238,257 @@ $ curl \
+**Format:** Integer indicating the number of seconds elapsed since January 1, 1970.
+
+`.ExpirationTime` - The time that the lease is set to expire. This time may be slightly earlier than the associated
+lease due to where this value is calculated compared with where Vault calculates details of the lease.<br />
+**Format:** 2006-01-02T15:04:05Z07:00 ([RFC3339](https://golang.org/pkg/time/#pkg-constants))
+
+`.ExpirationTimeSeconds` - The [unix time](https://en.wikipedia.org/wiki/Unix_time) the lease is set to expire. This
+time may be slightly earlier than the associated lease due to where this value is calculated compared with where Vault
+calculates details of the lease.<br />
+**Format:** Integer indicating the number of seconds elapsed since January 1, 1970.
+
+#### Username Template Fields
+
+The following parameters are available within the username template:
+
+**Important note:** If any of the following fields include dashes or underscores, they will not be removed/changed
+unless explicitly done so within the username template. For instance:
+
+If `RoleName` is `test-role` and the `username_template` is `v_{{.RoleName}}_{{unix_time}}`, the result of this template
+may be: `v_test-role_1234567890`. Note the `-` (dash) in the `test-role`. If the LDAP system Vault is managing
+restricts usernames/DNs to not allow for dashes (or other characters), the template must explicitly modify/remove those
+characters. In this example, the template can be changed to `v_{{.RoleName | replace "-" "_"}}` to replace
+the dashes with underscores. See [Template Functions](#template-functions) for more functions available.
+
+`.RoleName` - The name of the role the credentials are being generated from.
+
+`.DisplayName` - The display name associated with the user making the request against Value.
+
+#### Template Functions
+
+Both the LDIF templates as well as the username template support using the following functions:
+
+`random` - Generates a random string from lowercase letters, uppercase letters, and numbers. Must include a
+number indicating how many characters to generate.<br />
+**Example:** `{{random 20}}` generates 20 random characters
+
+`truncate` - truncates the previous value to the provided number of characters.<br />
+**Example:** `{{.FieldName | truncate 10}}`
+
+`truncate_sha256` - Truncates the previous value to the provided number of characters. The last 8 characters of the
+truncated value will be the first 8 characters of the SHA256 hash of the truncated characters.<br />
+**Example:** `{{.FieldName | truncate_sha256 20}}`. If `FieldName` is `abcdefghijklmnopqrstuvwxyz`, all characters after
+the 12th (`l`) are removed and SHA256 hashed (`872808ffbf0158b200d397a9884342497f23b153b1630229014cea1886ca6f20`.
+The first 8 characters of the hash (`872808ff`) are then appended to the end of the first 12 characters from the
+original value: `abcdefghijkl872808ff`.
+
+`uppercase` - Uppercases the provided value.
```

Comment:
Some of these end in `<br />`, but some don't?

Reply:
They all should to put the `Example` on the next line. Looks like I missed a couple =/